src/Behat/Config/Converter/CustomPrettyPrinter.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Behat\Config\Converter;
+
+use PhpParser\PrettyPrinter\Standard;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\MethodCall;
+
+class CustomPrettyPrinter extends Standard
```

Comment:
We override a couple of methods in the standard printer provided by the php parser library to customize the php output